DFC.ServiceTaxonomy.Editor.sln

```diff
@@ -57,6 +57,320 @@ EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "OrchardCore.AuditTrail", "OrchardCore.AuditTrail\OrchardCore.AuditTrail.csproj", "{C2C7C180-F824-4C79-A573-B4D639A4A562}"
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "OrchardCorePackages", "OrchardCorePackages", "{A7C90278-8F0E-4F99-8AD1-816B7C9FE739}"
+	ProjectSection(SolutionItems) = preProject
```

Comment:
We're up to rc10 now, and the rc9 packages have been deleted, so we don't want them as solution items.